config/webpack.config.js

```diff
@@ -231,7 +231,8 @@ module.exports = env => {
       host: devServer.host,
       port: devServer.port,
       headers: { 'Access-Control-Allow-Origin': '*' },
-      hot: true
+      hot: true,
+      disableHostCheck: true,
```

Comment:
@tbrisker I saw that the `disableHostCheck` option is not in the Procfile, is it ok that we're adding it?

Reply:
@laviro why is this change also required? I suppose it's ok since we don't necessarily have the host name configured properly in development and we don't really care about the hostnames in local development, but i might be missing something.

Reply:
Sorry for not explaining more.. this is something I also encountered when spinning up a new forklift katello-devel machine, and was missing in the configuration for some reason

Reply:
@laviro can you please update the commit message explaining what change and why the changes are needed? see https://chris.beams.io/posts/git-commit/ and especially point 7 there.

Reply:
@laviro IMHO that should be resolved, rather than worked around. Hosts should be correct and checked unless you can show that it really doesn't work. Usually it's a misconfiguration in your env.

Reply:
@ekohl in my case, I have created local aliases in `/etc/hosts` to my machines, since the actual name is too long and autogenerated.

Reply:
I also managed to fix it by adding the hostname to `config.hosts` via `config/environments/development.rb`
so I just removed it from our dev server config, although other devs might need to do the same step each time setting up a new environment.

Reply:
Can we document this somewhere? (In a different pr)

Reply:
There's a warning shown in the UI that you need to add the hostname to `config.hosts`,
usually, it's set up for the `fqdn` from your settings, see: https://github.com/theforeman/foreman/blob/develop/config/environments/development.rb#L63
so I am not sure we need documentation for it, maybe the warning that's shown is enough

Reply:
I just spun up a new `centos7-katello-devel-stable` and was unable to load any pages.  Adding my fqdn to `settings.yaml` didn't seem to do anything; it was only after adding back `disableHostCheck: true` here that I was able to get it working.